src/changelog.ts

```diff
@@ -39,7 +39,28 @@ export const generateChangelog: typeof def = async ({
 {{~/if}}
 {{~#if isPatch~}} </small> {{~/if}}
 `.trim();
-  preset.writer.mainTemplate! += "\n";
+  preset.writer.mainTemplate = `
+{{> header}}
+{{#if noteGroups}}
+{{#each noteGroups}}
+
+### ⚠ {{title}}
+
+{{#each notes}}
+* {{#if commit.scope}}**{{commit.scope}}:** {{/if}}{{commit.subject}} {{#if commit.hash}}([{{commit.shortHash}}](https://github.com/{{@root.owner}}/{{@root.repository}}/commit/{{commit.hash}})){{/if}}
+{{/each}}
+{{/each}}
+{{/if}}
+{{#each commitGroups}}
+
+{{#if title}}
+### {{title}}
+
+{{/if}}
+{{#each commits}}
+{{> commit root=@root}}
+{{/each}}
+{{/each}}\n`.trim();
```

Comment:
```suggestion
{{/each}}`.trim() + '\n';
```
I think `\n` needs to be outside otherwise it will be trimmed.